frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/pageComponents/PTSelectAssessmentYear.js

```
@@ -0,0 +1,129 @@
+        if (userType === "employee") {
+            const errorsPresent = !!Object.keys(localFormState.errors).length;
+            if (errorsPresent && !formState.errors?.[config.key]) setError(config.key, { type: "required" });
+            else if (!errorsPresent && formState.errors?.[config.key]) clearErrors(config.key);
+        }
+    }, [localFormState]);
+
+    if (userType === "employee") {
+        return (
+            <div style={{ marginLeft: "20px" }}>
+                <CardLabel style={{ marginBottom: "20px" }}>
+                    {t("Select Assessment Year") + " *"}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)